message_ix_models/model/water/data/water_for_ppl.py

```diff
@@ -544,7 +670,7 @@ def cool_tech(context: "Context") -> dict[str, pd.DataFrame]:
     )
     df_sw["time_dest"] = df_sw["time_dest"].astype(str)
     if context.nexus_set == "nexus":
-        out = pd.DataFrame()
+        # out = pd.DataFrame()
```

Comment:
If we don't need this line, please delete it :)

Reply:
Otherwise, please note why it's commented and when it should be uncommented again.